core/index.js

```diff
@@ -18,6 +20,13 @@ import {snapshotGather} from './gather/snapshot-runner.js';
 import {navigationGather} from './gather/navigation-runner.js';
 import * as LH from '../types/lh.js';
 
+// Needed for bundler.
+// eslint-disable-next-line no-undef
+if (!globalThis.process) {
+  // eslint-disable-next-line no-undef
+  globalThis.process = process;
+}
```

Comment:
Could this be injected by esbuild somehow? I would prefer not to add bundle specific code to regular files if we can.

Reply:
Could [`inject`](https://esbuild.github.io/api/#inject) work? If that doesn't work then I'm good just leaving it.